fixtures/projectWithBadImport/eslint-local-rules.js

```diff
@@ -0,0 +1,3 @@
+require('./does-not-exist');
```

Comment:
The `fixtures` directory looks like this:

```bash
├── fixtures
│   ├── projectWithBadImport
│   │   ├── a
│   │   └── eslint-local-rules.js
│   ├── projectWithNoResolution
│   ├── projectWithResolution
│   │   ├── a
│   │   └── eslint-local-rules.js
│   ├── projectWithResolutionCjs
│   │   ├── a
│   │   └── eslint-local-rules.cjs
│   └── projectWithResolutionIndex
│       ├── a
│       └── eslint-local-rules
│           └── index.js
```

New directory structures can be created in here when adding more tests